hypervisor/arch/x86/page.c

```diff
@@ -199,9 +199,15 @@ void *get_reserve_sworld_memory_base(void)
 	return post_uos_sworld_memory;
 }
 
-static inline bool large_page_not_support(__unused enum _page_table_level level)
```

Comment:
/*
 * Pages without execution right, such as MMIO, can always use large page base on hardware capability,, even if the VM is an RTVM. This can save page table page # and improve TLB hit rate.
 * /